src/atomate2/vasp/sets/core.py

```diff
@@ -800,6 +808,123 @@ def get_kpoints_updates(
         return {"reciprocal_density": self.reciprocal_density}
 
 
+@dataclass
+class MDSetGenerator(VaspInputGenerator):
+    """
+    Class to generate VASP molecular dynamics input sets.
+
+    Parameters
+    ----------
+    ensemble
+        Molecular dynamics ensemble to run. Options include `nvt`, `nve`, and `npt`.
+    start_temp
+        Starting temperature. The VASP `TEBEG` parameter.
+    end_temp
+        Final temperature. The VASP `TEEND` parameter.
+    nsteps
+        Number of time steps for simulations. The VASP `NSW` parameter.
+    time_step
+        The time step (in femtosecond) for the simulation. The VASP `POTIM` parameter.
+    **kwargs
+        Other keyword arguments that will be passed to :obj:`VaspInputGenerator`.
+    """
+
+    ensemble: str = "nvt"
+    start_temp: float = 300
+    end_temp: float = 300
+    nsteps: int = 1000
+    time_step: int = 2
+    auto_ispin: bool = True
+
+    def get_incar_updates(
+        self,
+        structure: Structure,
+        prev_incar: dict = None,
+        bandgap: float = 0,
+        vasprun: Vasprun = None,
+        outcar: Outcar = None,
+    ) -> dict:
+        """
+        Get updates to the INCAR for a molecular dynamics job.
+
+        Parameters
+        ----------
+        structure
+            A structure.
+        prev_incar
+            An incar from a previous calculation.
+        bandgap
+            The band gap.
+        vasprun
+            A vasprun from a previous calculation.
+        outcar
+            An outcar from a previous calculation.
+
+        Returns
+        -------
+        dict
+            A dictionary of updates to apply.
+        """
+        updates = self._get_ensemble_defaults(structure, self.ensemble)
+
+        # Based on pymatgen.io.vasp.sets.MPMDSet.
+        updates.update(
+            {
+                "ENCUT": 520,
+                "TEBEG": self.start_temp,
+                "TEEND": self.end_temp,
+                "NSW": self.nsteps,
+                "POTIM": self.time_step,
+                "LCHARG": False,
+                "LPLANE": False,
+                "LWAVE": True,
```

Comment:
Does this write the wavecar after each step? Could this be very expensive due to IO time of writing supercell wave functions?